deploy/FlapperInit.sol

```diff
@@ -48,12 +49,41 @@ interface PairLike {
     function token1() external view returns (address);
 }
 
+interface SplitterLike {
+    function vat() external view returns (address);
+    function daiJoin() external view returns (address);
+    function farm() external view returns (address);
+    function rely(address) external;
+    function file(bytes32, uint256) external;
+    function file(bytes32, address) external;
+}
+
+interface FarmLike {
+    function setRewardsDistribution(address) external;
+    function setRewardsDuration(uint256) external;
+}
+
+interface FlapperCaller {
+    function file(bytes32, address) external;
+}
+
 struct FlapperUniV2Config {
     uint256 hop;
     uint256 want;
     address pip;
     uint256 hump;
     uint256 bump;
+    address pair;
+    address caller;
+    bytes32 chainlogKey;
+}
+
+struct SplitterConfig {
```

Comment:
We could consider to just have the `hop` value and set directly that to the rewardsDistribution as well.